app/src/main/java/com/scrumbums/donationboi/controllers/ItemSearchActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.scrumbums.donationboi.controllers;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;import java.util.function.Predicate;
+
+import android.widget.Spinner;
+
+import com.scrumbums.donationboi.R;
+import com.scrumbums.donationboi.model.entities.Item;
+import com.scrumbums.donationboi.model.entities.Store;
+import com.scrumbums.donationboi.model.util.Converters;
+import com.scrumbums.donationboi.model.util.DatabaseAbstraction;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class ItemSearchActivity extends AppCompatActivity {
+
+    private Button searchButton;
+    private Spinner categorySpinner;
+    private EditText searchBar;
+    private ListView results;
+    private ArrayAdapter adapter;
+    private List<Item> inventoryArray;
+    private List<Item> filteredArray;
+    private Snackbar notFound;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        final Intent intent = getIntent();
+
+        int storeId = intent.getIntExtra("storeId", 0);
+        if (storeId == 0) {
+            inventoryArray = new ArrayList<>();
+            for(Store s: DatabaseAbstraction.getStoresArrayList()) {
+                inventoryArray.addAll(s.getInventoryArrayList());
+            }
+        } else {
+            inventoryArray = DatabaseAbstraction.getStore(storeId).getInventoryArrayList();
+        }
+        setContentView(R.layout.item_search);
+
+        notFound = Snackbar.make(findViewById(R.id.search_constraint_layout), "No Items Found", 100);
```

Comment:
I think the 100 here is setting the time the Snackbar stays on screen?  If so, can you change it to stay on screen for like 3 seconds or something?

Reply:
oh yeah totally meant to up that and forgot thanks